adyenofficial.php

```diff
@@ -325,6 +328,63 @@ private function createAdyenDatabaseTables()
             $this->createAdyenNotificationTable();
     }
 
+    /**
+     * Copy waiting_for_payment email template to all /mails/ subdirectories
+     * Required until https://github.com/PrestaShop/PrestaShop/issues/24336 is implemented
+     *
+     * @return bool
+     */
+    public function copyEmailTemplates()
+    {
+        $allDirectories = true;
```

Comment:
This naming is a bit confusing, could be maybe `$successfulCopy`

Reply:
https://github.com/Adyen/adyen-prestashop/pull/195/commits/8c34dde4a8600fc3e1ced7944cd81bd8517a9247